charts/unleash-edge/README.md

```diff
@@ -0,0 +1,59 @@
+# Unleash Edge
+
+Unleash Edge offers a way to use Unleash in client-side applications, such as single page and native apps. It can also be used for server-side applications, taking the place of the main Unleash server. This has the added benefit of not triggering database operations for every client refresh.
+
+See the [reference documentation](https://docs.getunleash.io/reference/unleash-edge) for more on the reasons why you might want to deploy Edge in your cluster.
+
+This chart bootstraps an [Unleash Edge](https://github.com/Unleash/unleash-edge) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+No extra configuration is needed besides pointing edge to an Upstream URL.
+
+## Prerequisites
+
+- Kubernetes 1.23+
+- Helm 3+
+
+## Get Repository Info
+
+```console
+helm repo add unleash https://docs.getunleash.io/helm-charts
+helm repo update
+```
+
+_See [the helm repository docs](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+helm install unleash-edge unleash/unleash-edge
```

Comment:
Should this be `unleashorg/unleash-edge`? Just based on this: https://github.com/Unleash/helm-charts/pull/67/files#diff-d2fc3416f6ac8a6efde1032344d7d971f1e82003f21bdd03ffe2c87e9aea14c7R9

Reply:
No. It's a fair question, but that address has nothing to do with the docker image (which is the unleashorg/) and everything to do with the `repo add` command run earlier. We ran it with the argument `helm repo add unleash https://...` where the first argument to add is the name we want our helm repository to have.